src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -5,6 +5,22 @@
       "toolInfo": {
         "name": "AndroidStudio"
       },
+      "runInfo": {
+        "fileInfo": {
+          "app/src/androidTest/java/microsoft/androidstudioformatexample/ApplicationTest.java": [
+            {
+              "uri": "app/src/androidTest/java/microsoft/androidstudioformatexample/ApplicationTest.java",
```

Comment:
Why do we recapitulate the uri in its entirety when it doesn't differ from the key? i.e., in cases when the fileReference array consists of a single item, uri could be optional. ?